Wall/ColorText.hpp

```diff
@@ -2,7 +2,7 @@
 //  ColorText.hpp
 //  Wall
 //
-//  Created by Nicolas Fricker on 6/16/20.
+//  Created by Fricker95 on 6/16/20.
```

Comment:
**Adding Features** — Oh really, now?  Are you iseekwonderful?  Orinion?  Andre Kalisch?  I see no evidence in the commit history to warrant rewriting history as a "feature".

Reply:
Me.....?.?.

Reply:
No I am not, mad respect to them though. I did write the entirety of this file and if manigamer22 wants to pull request that into iseekwonderful's master, I want to remove my name for privacy reasons. Whats wrong with using a pseudonym?